config/initializers/content.rb

```diff
@@ -0,0 +1,16 @@
+require 'yaml'
+
+# ERB.new allows us to use ERB tags in the YAML
```

Comment:
This is a good comment - seems that in the future if we do not need ERB tags, then we can simplify this with just a `YAML.load_file()`.